ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>pipeline_root=PIPELINE_ROOT,</p><p>)</p><p> </p><p>Drop the trailing comma.</p><p> </p><p> </p><p>&nbsp;&nbsp;&nbsp;&nbsp;two_outputs_task.outputs["output_two"],</p><p>&nbsp;&nbsp;)</p><p> </p><p>Ditto.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>